server_fn_macro/src/lib.rs

```diff
@@ -119,6 +119,7 @@ pub fn server_macro_impl(
         res_ty,
         client,
         custom_wrapper,
+        impl_from: impl_into,
```

Comment:
This line is what made me think it should be renamed: if what it does is `impl From<_>`, and we're using that language internally, it might make sense to do the same thing in the API.

Reply:
The funny thing is that initially it was named actually as `impl_from`, but later on I renamed it to be `impl_into` -- this line of code seems to be an artifact of that process. I've changed it back in the receing commit.